unit tests inconsistent in src/pkg/packager

### Describe what should be investigated or refactored
unit tests in src/pkg/packager run inconsistently. Running on ubuntu linux go1.21.3 
Sometimes when running go test in src/pkg/packager you tests pass. 
Other times you get. Seemingly around 50/50
```bash
--- FAIL: TestValidateLastNonBreakingVersion (0.00s)
    --- FAIL: TestValidateLastNonBreakingVersion/invalid_semantic_version_(CLI_version) (0.00s)
        common_test.go:231: 
                Error Trace:    /home/austin/code/zarf/src/pkg/packager/common_test.go:231
                Error:          []string{"CLIVersion is set to 'this shouldn't get evaluated when the lastNonBreakingVersion is empty' which can cause issues with package creation and deployment. To avoid such issues, please set the value to the valid semantic version for this version of Zarf."} does not contain "CLIVersion is set to 'invalidSemanticVersion' which can cause issues with package creation and deployment. To avoid such issues, please set the value to the valid semantic version for this version of Zarf."
                Test:           TestValidateLastNonBreakingVersion/invalid_semantic_version_(CLI_version)
```

--- a/src/pkg/packager/common_test.go
+++ b/src/pkg/packager/common_test.go
@@ -207,8 +207,6 @@ func TestValidateLastNonBreakingVersion(t *testing.T) {
 		testCase := testCase
 
 		t.Run(testCase.name, func(t *testing.T) {
-			t.Parallel()
-
 			config.CLIVersion = testCase.cliVersion
 
 			p := &Packager{

